pkg/codegen/go/gen.go

```diff
@@ -1352,7 +1541,7 @@ func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error
 		// Run Go formatter on the code before saving to disk
 		formattedSource, err := format.Source([]byte(contents))
 		if err != nil {
-			panic(errors.Errorf("invalid Go source code:\n\n%s", contents))
+			panic(fmt.Errorf("invalid Go source code:\n\n%s: %s: %w", relPath, contents, err))
```

Comment:
Prefer `errors.Errorf`

```suggestion
			panic(errors.Errorf("invalid Go source code:\n\n%s: %s: %w", relPath, contents, err))
```